packages/evo-marko/src/tags/evo-accordion/index.marko

```diff
@@ -0,0 +1,67 @@
+import type { Input as DetailsInput } from "<evo-details>";
+export interface Input extends Omit<Marko.HTML.UL, `on${string}`> {
+    size?: "regular" | "large";
+    autoCollapse?: boolean;
+    a11yText?: string;
+    details?: Marko.AttrTag<
+        Omit<DetailsInput, "size" | "alignment" | "open" | "openChange">
+    >;
+    open?: number | number[];
+    openChange?: (open: number | number[]) => void;
+}
+
+<const/{
+    class: inputClass,
+    size,
+    details,
+    autoCollapse = false,
+    a11yText,
+    open: inputOpen,
+    openChange,
+    ...htmlInput
+}=input>
+<const/items=[...(details || [])]>
+
+<let/open:=input.open>
+
+<ul
+    class=["accordion", size === "large" && "accordion--large", inputClass]
+    aria-roledescription=(a11yText ?? "accordion")
+    ...htmlInput>
+    <for|item, index| of=items>
+        <li>
+            <evo-details
+                ...item
+                open=(
+                    Array.isArray(open) ? open.includes(index) : open === index
+                )
+                openChange(isOpen) {
+                    if (autoCollapse) {
```

Comment:
This test should be 
```
if (!Array.isArray(open))
```
So if they pass in `open` as an array, treat it as a multi select and they can have multiple sections open. 
If they pass in `open` as a number, then only 1 section can be opened at a time (IE autoCollapse is enabled)